crates/frontend/src/utils.rs

```diff
@@ -516,3 +516,14 @@ pub fn set_local_storage(_key: &str, _value: &str) -> Option<()> {
         }
     }
 }
+
+pub fn get_key_type(schema: &Map<String, Value>) -> String {
+    if schema.contains_key("enum") {
+        String::from("ENUM")
+    } else {
+        match schema.get("type").unwrap_or(&Value::Null) {
```

Comment:
Why are we using unwrap and match both here ? 
Possible to change it to : 
`match schema.get("type") {
    Some(Value::String(str_)) => str_.to_ascii_uppercase(),
    _ => String::from("STRING"),
}
`